src/locale/locales/es/messages.po

```diff
@@ -986,31 +986,31 @@ msgstr "Bluesky"
 
 #: src/view/com/auth/server-input/index.tsx:151
 msgid "Bluesky is an open network where you can choose your hosting provider. If you're a developer, you can host your own server."
-msgstr ""
+msgstr "Bluesky es una red abierta donde puede elegir su proveedor de alojamiento.Si eres un desarrollador, puedes alojar tu propio servidor."
```

Comment:
`Bluesky es una red abierta donde puede elegir su proveedor de alojamiento. Si eres un desarrollador, puedes alojar tu propio servidor.`

(missing space after the .)

Reply:
Fixed